docs/app/data/components-page.json

```diff
@@ -1429,16 +1429,41 @@
                     "version": "AngularJS",
                     "deprecated": true
                 },
+                {
+                    "title": "Sorting",
```

Comment:
Could we call this "Toolbar Sorting" to give it some context compared to column sorting? And use the link functionality from the deprecated sections.